### PR TITLE
[DPE-3023] Add badges to README.md

### PR DESCRIPTION
**Pull request by deusebio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
-# charmed-kafka-snap
+# Charmed Kafka snap repository.
```
Comment (zmraul) [inline README.md:1]: Nice

File: README.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 
-Charmed Kafka snap repository.
-
-This snap is based on the kafka snap: https://git.launchpad.net/kafka-snap/
+[![CharmHub Badge](https://snapcraft.io/charmed-kafka/badge.svg)](https://snapcraft.io/charmed-kafka)
+[![Release](https://github.com/canonical/charmed-kafka-snap/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/charmed-kafka-snap/actions/workflows/publish.yaml)
```
Comment (zmraul) [inline README.md]: We have `build & test` CI. I think that could be added here.

**Review by zmraul — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (deusebio) [reply README.md]: I believe the problem here is that we don't have the "build & test" in the release pipeline (although maybe we should) but only in the PRs. So the workflows only include feature branches, see [here](https://github.com/canonical/charmed-kafka-snap/actions/workflows/ci.yaml)

File: README.md [hunk 3]
```diff
@@ -1,16 +1,15 @@
-# charmed-kafka-snap
+# Charmed Kafka snap repository.
 
-Charmed Kafka snap repository.
-
-This snap is based on the kafka snap: https://git.launchpad.net/kafka-snap/
+[![CharmHub Badge](https://snapcraft.io/charmed-kafka/badge.svg)](https://snapcraft.io/charmed-kafka)
```
Comment (delgod) [inline README.md]:
```suggestion
[![Snapcraft Badge](https://snapcraft.io/charmed-kafka/badge.svg)](https://snapcraft.io/charmed-kafka)
```

File: README.md [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
+[![Release](https://github.com/canonical/charmed-kafka-snap/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/charmed-kafka-snap/actions/workflows/publish.yaml)
```
Comment (delgod) [inline README.md:5]: [Update Available](https://github.com/canonical/charmed-kafka-snap/actions/workflows/on_kafka_update_available.yaml)
```suggestion
[![Update Available](https://github.com/canonical/charmed-kafka-snap/actions/workflows/on_kafka_update_available.yaml/badge.svg)](https://github.com/canonical/charmed-kafka-snap/actions/workflows/on_kafka_update_available.yaml)
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (deusebio) [reply README.md:5]: Uhm, I would skip this one if you don't mind, as we should actually remove that workflow. That workflow was actually looking upstream, but we now build from source.

**Review by zmraul — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Pull request merged**